app/templates/partials/footer2.html

```diff
@@ -54,7 +54,7 @@
                             <a class="footer2__link" href="https://www.ons.gov.uk/news" target="_blank">News</a>
                         </li>
                     </ul>
-                    </br>
+                    <br />
```

Comment:
I think we can just remove this `<br />` altogether. Makes no difference visually